temporal-sdk/src/main/java/io/temporal/internal/worker/SingleWorkerOptions.java

```diff
@@ -126,35 +126,33 @@ public Builder setDefaultHeartbeatThrottleInterval(Duration defaultHeartbeatThro
     }
 
     public SingleWorkerOptions build() {
+      PollerOptions pollerOptions = this.pollerOptions;
       if (pollerOptions == null) {
-        pollerOptions =
-            PollerOptions.newBuilder()
-                .setPollBackoffInitialInterval(Duration.ofMillis(200))
```

Comment:
This code path is dead. We come here only in some tests where we do
```
        SingleWorkerOptions.newBuilder().build()
```

and these options and not important for these tests. As a result, these magic values 200 and 20 here are never used and beter be removed from the production code paths.